invenio_rdm_records/theme/assets/scss/invenio_rdm_records/variables.scss

```diff
@@ -0,0 +1,13 @@
+/*
+ *   Copyright (C) 2020 CERN.
+ *   Copyright (C) 2020 Northwestern University.
+ *
+ * Invenio RDM Records is free software; you can redistribute it and/or modify
+ * it under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+
+$brand-primary: #0377cd;
+
+// This is nowhere in Zenodo nor Invenio (?)
+$font-size-base: 14px;
```

Comment:
just shows up in bootstrap for me...

Reply:
Yup, in Zenodo it's a variable but not defining it was failing to me (even when importing bootstrap). Since we are moving to Semantic-UI I'll just leave it as is for now if it's okay for you

Reply:
Sure.